scripts/zones/Waughroon_Shrine/mobs/Osschaarts_Automaton.lua

```diff
@@ -0,0 +1,11 @@
+-----------------------------------
```

Comment:
There's no logic in this file so is it needed? Might be able to just delete this whole file. Same with the other pet luas other than the avatar one.